readthedocs/doc_builder/environments.py

```diff
@@ -150,21 +148,16 @@ def run(self):
         if self.combine_output:
             stderr = subprocess.STDOUT
 
-        environment = {}
-        environment.update(self.environment)
-        environment['READTHEDOCS'] = 'True'
-        if self.build_env is not None:
-            environment['READTHEDOCS_VERSION'] = self.build_env.version.slug
-            environment['READTHEDOCS_PROJECT'] = self.build_env.project.slug
-            environment['READTHEDOCS_LANGUAGE'] = self.build_env.project.language
+        environment = self.environment.copy()
         if 'DJANGO_SETTINGS_MODULE' in environment:
             del environment['DJANGO_SETTINGS_MODULE']
         if 'PYTHONPATH' in environment:
             del environment['PYTHONPATH']
+
+        env_paths = os.environ.get('PATH', '').split(':')
```

Comment:
Why did we change this logic? If we don't set `bin_path`, isn't this just setting `PATH` to the same thing as before?

Reply:
Yeah, we copy the path from the host and add any additional `bin_path`. This is being added here because it was removed from https://github.com/readthedocs/readthedocs.org/pull/6482/files#r363843701. So this only executes in a local environment, not on both (docker and local, as the previous code to this change)

Reply:
This could use a comment then, since it's not obvious why we're doing this. 

```suggestion
        # Always copy the PATH from the host into the environment
        env_paths = os.environ.get('PATH', '').split(':')
```